nfs-ganesha/config/definitions/nfs-ganesha.yml

```diff
@@ -61,13 +61,9 @@ If this is checked, then the binaries will be built and pushed to chacra even if
           type: label-expression
           name: AVAILABLE_DIST
           values:
-            - centos6
```

Comment:
@alfredodeza were these values necessary? I'm confused about which distros get triggered to build when I push the "Build" button in "Build With Parameters" section of the jenkins UI.